iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -198,6 +198,10 @@ TEST_F(PoshRuntime_test, GetMiddlewareInterfaceIsSuccessful)
     EXPECT_EQ(true, interfacePortData->m_doInitialOfferForward);
 }
 
+TEST_F(PoshRuntime_test, GetMiddlewareInterfaceIsNotSuccessful)
```

Comment:
I would adjust the name so that it is clear under which condition it is not successful